src/modules/joke.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const getJoke = async () => {
+  let currentJoke = await axios.get("/jokeGetter");
```

Comment:
you are making a request here, so please set the variable to response instead of currentJoke